example/components/Example.jsx

```diff
@@ -1,12 +1,13 @@
 import React, { PropTypes } from 'react'
 import ENV from '../env.json'
+import image from '../logo.svg'
 import '../styles/example.scss'
 
 class Example extends React.Component {
   render () {
     return (
       <div className='example'>
-        <img src='http://bit.ly/1YEe536'/>
+        <div style={{width: '100px'}} dangerouslySetInnerHTML={{__html: image}} />
```

Comment:
This seems the be the way the React folks think you should implement `.svg` files.